packages/core/content-releases/server/src/services/release.ts

```diff
@@ -35,15 +39,49 @@ const createReleaseService = ({ strapi }: { strapi: LoadedStrapi }) => ({
       },
     });
   },
-  findOne(id: GetRelease.Request['params']['id'], query = {}) {
-    return strapi.entityService.findOne(RELEASE_MODEL_UID, id, query);
-  },
-  findMany(query?: GetReleases.Request['query']) {
-    return strapi.entityService.findMany(RELEASE_MODEL_UID, {
-      ...query,
+  findManyForContentTypeEntry(
+    contentTypeUid: GetContentTypeEntryReleases.Request['query']['contentTypeUid'],
+    entryId: GetContentTypeEntryReleases.Request['query']['entryId'],
+    {
+      hasEntryAttached,
+    }: { hasEntryAttached?: GetContentTypeEntryReleases.Request['query']['hasEntryAttached'] } = {
```

Comment:
what do you think about make this more general? We can make contentTypeUid and entryId optionals (also add them to the same object where hasEntryAttached lives) and change the name of this to: `findManyNotReleased`

Reply:
I'm not necessarily opposed but I'm wondering:
- Will we ever need to get an unpaginated list of all releases not released? We can already get that paginated list with findPage.
- How would we handle it in the controller. Currently I'm using the existing findMany controller with a condition. But I think this new approach might require its own route/controller.

Reply:
I pushed another commit, I think we will need this specific service for getting releases for entries.